service/src/main/java/bio/terra/externalcreds/ExternalCredsCronApplication.java

```diff
@@ -39,12 +39,21 @@ public ExternalCredsCronApplication(ProviderService providerService) {
     this.providerService = providerService;
   }
 
-  @Scheduled(fixedRateString = "#{${externalcreds.background-job-interval-mins} * 60 * 1000}")
+  @Scheduled(
+      fixedRateString = "#{${externalcreds.refresh-background-job-interval-mins} * 60 * 1000}")
   public void checkForExpiringCredentials() {
     log.info("beginning checkForExpiringCredentials");
     var expiringPassportCount = providerService.refreshExpiringPassports();
     log.info(
         "completed checkForExpiringCredentials",
         Map.of("expiring_passport_count", expiringPassportCount));
   }
+
+  @Scheduled(
+      fixedRateString = "#{${externalcreds.validation-background-job-interval-mins} * 60 * 1000}")
+  public void validateVisas() {
+    log.info("beginning validateVisas");
+    var checkedPassportCount = providerService.validatePassportsWithAccessTokenVisas();
+    log.info("completed validateVisas", Map.of("checked_passport_count", checkedPassportCount));
+  }
```

Comment:
I think we want another scheduled function, just put it in `checkForExpiringCredentials`

Reply:
Are we not running those at different intervals? Or is there a way to do that within the same function?

Reply:
my main concern is concurrency between the 2 jobs, I think it makes sense to first make sure we refresh anything that is going to expire soon then check of any visas that need to be validated

Reply:
I don't think the intervals for running the background job need to be different. Each task has a different threshold for things to refresh/validate and the background job interval is how often we check for things that hit the threshold.

Reply:
In that case should we remove the separate config variable for the frequency of the validation job, or just leave it for now but use the refresh config interval for both?

Reply:
remove it